docs/python/python-quick-start.md

```diff
@@ -0,0 +1,125 @@
+
+### Clone a Python project from version control
+
+If you have an existing Python project in version control, you can clone it locally by selecting **Clone Git Repository** in the VS Code Welcome page, or by selecting **Clone Repository** in the Explorer panel.
+
+## UI Tour
+
+When you launch VS Code for the very first time, you will need to install the Python extension to get full Python support and UI. Let’s look at the UI once the Python extension is installed.
+
+![Image of the Python UI highlights in VS Code.](/docs/python/images/quick-start/ui-tour.png)
+
+## Code Actions
+
+Code Actions (also known as Quick Fixes) are provided by Warnings or Errors to help fix issues you may be experiencing in your code. These helpful hints are displayed in the editor left margin as a lightbulb (💡). Click on the light bulb to display Code Action options which can come from extensions such as Python, Pylance, or VS Code itself. For more information about Code Actions, see [Python Quick Fixes](https://code.visualstudio.com/docs/python/editing#_quick-fixes).
+
+![Gif showing Code Actions in a Python project.](images/editing/quickFix.gif)
+
+## Python Commands
+
+Python commands can be accessed through the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) (`kb(workbench.action.showCommands)`). From the Command Palette, you have access to the full functionality of VS Code, but for Python specific functionality begin typing **“Python: “** to locate the commands accessible through the Python extension.
+
+![Gif demonstrating how to access Python commands in the Command Palette.](/docs/python/images/quick-start/cmd-plt-python.gif)
+
+## Run, debug, and test
+
+Now that you are more familiar with Python in VS Code, let’s learn how to run, debug, and test your code.
+
+### Run
+
+There are a few ways to run Python code in VS Code.
+
+To run the active Python file, click the **Run Python File in Terminal** play button in the top-right side of the editor.
+
+![Image showing the Run Python File in Terminal play button.](images/tutorial/run-python-file-in-terminal-button.png)
+
+There are three other ways you can run Python code within VS Code:
+-	Right-click anywhere in the editor window and select **Run > Python File** in Terminal (which saves the file automatically):
+![Image showing the Python run options in the context menu.](images/tutorial/run-python-file-in-terminal.png)
+-	Select one or more lines, then press `kbstyle(Shift+Enter)` or right-click and select **Run Selection/Line in Python Terminal**. This command is convenient for testing just a part of a file.
+-	From the Command Palette (`kb(workbench.action.showCommands)`), select the **Python: Start REPL** command to open a REPL terminal for the currently selected Python interpreter. In the REPL, you can then enter and run lines of code one at a time.
```

Comment:
```suggestion
There are also additional ways you can iteratively run snippets of your Python code within VS Code:
-	Select one or more lines, then press `kbstyle(Shift+Enter)` or right-click and select **Run Selection/Line in Python Terminal**. This command is convenient for testing just a part of a file.
-	From the Command Palette (`kb(workbench.action.showCommands)`), select the **Python: Start REPL** command to open a REPL terminal for the currently selected Python interpreter. In the REPL, you can then enter and run lines of code one at a time.
```

Reply:
technically there are more than just 3 additional ways :p but I figured it was worth covering run selection and the REPL case, and not mention the one from the context menu since it does pretty much the same as the run button